hpx/util/itt_notify.hpp

```diff
@@ -117,12 +122,14 @@ HPX_EXPORT void itt_thread_set_name(char const*);
 HPX_EXPORT void itt_thread_ignore();
 
 HPX_EXPORT void itt_task_begin(___itt_domain const*, ___itt_string_handle*);
+HPX_EXPORT void itt_task_begin(___itt_domain const*, ___itt_id*,
+    ___itt_string_handle*);
 HPX_EXPORT void itt_task_end(___itt_domain const*);
 
 HPX_EXPORT ___itt_domain* itt_domain_create(char const*);
-HPX_EXPORT ___itt_string_handle* itt_task_handle_create(char const*);
+HPX_EXPORT ___itt_string_handle* itt_string_handle_create(char const*);
```

Comment:
None we could check at compile time. The Intel itt_notify.h has the same version number in it for years, even while the API itself has changed significantly.